.github/workflows/prChecks.yml

```diff
@@ -0,0 +1,26 @@
+name: PR checks
+run-name: Checking PR created by ${{ github.actor }}
+on: [push]
+jobs:
+  Test-PR:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v4
+      - name: Setup Java
+        uses: actions/setup-java@v4
+        with:
+          java-version: '21'
+          distribution: 'temurin'
+
+      - name: Setup Gradle
+        uses: gradle/actions/setup-gradle@v3
+        with:
+          build-scan-publish: true
+          build-scan-terms-of-use-url: "https://gradle.com/terms-of-service"
+          build-scan-terms-of-use-agree: "yes"
+
+      - name: Build with Gradle
+        run: ./gradlew build
+
+
```

Comment:
please add a step to attach the test results to the github action run.

There are a number of 3rd party actions available that do that.
I am aware that the test results are also available in the gradle build scan but I think it's good to have some info available in the GH ui.